test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -97,6 +97,8 @@ def test_tensorflow_with_horovod_gpu(
         pytest.skip(
             f"Image {tensorflow_training} is incompatible with instance type {ec2_instance_type}"
         )
+    if "2.13" in tensorflow_training:
```

Comment:
horovod test should be skipped for TF2.13 and above. Lets declare a common function to be used for all the test or refer other test for similar implementation

Reply:
Made use of an existing function for this operation.

Reply:
This is updated thru a fixture and is operating expected.